.changeset/forty-hounds-work.md

```diff
@@ -0,0 +1,53 @@
+---
+"@comet/blocks-api": major
+"@comet/cms-api": major
+---
+
+Support "real" dependency injection in `BlockData#transformToPlain`
```

Comment:
@nsams @thomasdax98 should I list all breaking changes here? Doing so would result in a rather large changeset.

Reply:
Maybe in the migration guide? https://github.com/vivid-planet/comet/pull/1946/commits/95158fdb1783da1373422fc1c968ad5106ce0db4

Reply:
I could also split it into multiple changesets for the separate changes (which are "only" related)

Reply:
I've extended the changeset to list the breaking changes and link to the migration: 9321447d1ea05039ea702203ee1df47c4ed149a2. Please review.